.changeset/curly-beers-raise.md

```diff
@@ -0,0 +1,19 @@
+---
+'@astrojs/compiler': minor
+---
+
+The compiler will now return `errors` and `warnings` which can be handled by the consumer. For example:
+
+```js
+import { transform } from '@astrojs/compiler';
+
+const { errors } = await transform(file, opts);
+for (const error of errors) {
+  console.error(error.text);
+}
+// or
+const { code, warnings } = await transform(file, opts);
+for (const warning of warnings) {
+  console.warn(warning.text);
+}
+```
```

Comment:
Not a big deal, but the changeset is outdated for the new return shape

Reply:
Great catch, thank you!